tests/conftest.py

```diff
@@ -40,6 +41,11 @@ def lakefs_options() -> LakeFSOptions:
     )
 
 
+@pytest.fixture
+def fs(lakefs_options: LakeFSOptions) -> LakeFSFileSystem:
+    return LakeFSFileSystem(**lakefs_options._asdict())
```

Comment:
Might have been better on my part to make it a dataclass and use `asdict` here to save an underscored access, oh well.

Reply:
Made `LakeFSOptions` a dataclass in 4e16c07b50d4051f3e9b4563b1a9bc8142beb6c5